app/models/solidus_subscriptions/checkout.rb

```diff
@@ -117,6 +120,10 @@ def ship_address
       subscription.shipping_address || user.ship_address
     end
 
+    def bill_address
```

Comment:
I think billing_address would be a bit more clear

Reply:
for whatever reason, `bill_address` and `ship_address` are the methods already attached to the user, so these match the values already in place for solidus